src/components/Image/index.tsx

```diff
@@ -59,12 +61,14 @@ function Image({source: propsSource, isAuthTokenRequired = false, session, onLoa
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [propsSource, isAuthTokenRequired]);
 
+    const shouldOpacityBeZero = isObjectPositionTop && !aspectRatio;
```

Comment:
Could you clarify how this change relates to the issue?

Reply:
Gotcha, thanks for the explainer! Could you add a brief comment to explain `shouldOpacityBeZero`? It's a bit hard to understand what's going on otherwise.